recipe/meta.yaml

```diff
@@ -195,8 +195,7 @@ outputs:
         - poppler
       run:
         - libgdal-core >={{ ".".join(version.split(".")[:2]) }}
-        # temp until poppler run_export takes hold
-        - poppler >=24.08,<24.09
+        - poppler
```

Comment:
I think this can and should be removed from `run` now.  We want poppler's own `run_exports` to add it to `run` with the appropriate constraints.  I think there will be a warning about over-depending with it still here.

Reply:
Sounds right, will do once the rerender finishes.

Reply:
(I'm writing like I'm confident about these things but I'm also pretty hazy on this stuff.)